### ISSUE TITLE
Bug - TextToSpeech conversion of results - spellingTemplateApp

### ISSUE THREAD

**Issue by jahandaniyal** [BuildmLearn/Toolkit-mobile-templates#17]
The announcer pauses at wrong instances, making it hard to follow the no. of correct/wrong and unanswered questions.
The result is announced like - 
Correct Spelled    "pause"      Two Wrong Spelled     "pause"     Three unanswered    "pause"     One

Instead of - Correct spelled    "pause"      Two     "long pause" 
Wrong Spelled     "pause"     Three       "long pause"
unanswered     "pause"    One

Bug Fixes that I have made :-
1. Corrected the above issue.
2. Corrected "msg_txt" in styles.xml - Changed  "SPEAK" to "LISTEN" -  (To correspond to the button "LISTEN", the user is not using speech instead they are listening to the words)
3. Code enchancement - 

String speechText = getString(R.string.correct) + " " + correct
				+ getString(R.string.wrong_spelled) + " "
									+ wrong + getString(R.string.unanswered)
									+ " " + unanswered;
				
( string has already been extracted earlier in the code, so we can simply use getText() method to re-use the extracted string instead of using so many concatenations -  Saves few lines of code and reduces compute time)
					
String speechText =  (String) mTv_Correct.getText()+ "." + (String)mTv_Wrong.getText()+ "." + (String)mTv_Unanswered.getText();

Should I create a pull-request ?   or should I wait for approval of these bugs first?
This will be my first contribution to this project, please let me know if there's some procedure I need to follow.

Regards,

**Comment by opticod** [comment]
Solve each bug issue in a particular PR. you can proceed fixing them :+1: @jahandaniyal

**Comment by opticod** [comment]
the app has been redesigned and fixed.
